src/wallet/wallet.cpp

```diff
@@ -1549,6 +1549,18 @@ bool CWallet::IsSproutNullifierFromMe(const uint256& nullifier) const
     return false;
 }
 
+bool CWallet::IsSaplingNullifierFromMe(const uint256& nullifier) const
+{
+    {
```

Comment:
Question: Is there a reason for the extra indentation here?

Reply:
Inherited style.  The scope in the braces is the lifetime of the LOCK.